qemu.go

```diff
@@ -327,22 +327,26 @@ func networkModelToQemuType(model NetInterworkingModel) ciaoQemu.NetDeviceType {
 	}
 }
 
-func (q *qemu) appendNetworks(devices []ciaoQemu.Device, endpoints []Endpoint) []ciaoQemu.Device {
-	for idx, endpoint := range endpoints {
+var networkIndex = 0
+
+func (q *qemu) appendNetwork(devices []ciaoQemu.Device, endpoint Endpoint) []ciaoQemu.Device {
+	virtualEndpoint, ok := endpoint.(*VirtualEndpoint)
```

Comment:
Again, we should switch on the `Endpoint` type to make the code more generic and understandable.